src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './StudentCoordinatorSettings.css';
+import OrganizationDetails from './OrganizationDetails';
+import AccountManager from './AccountManager';
+import { AccountType } from '../../Enums';
+import MemberAccount from './MemberAccount';
+import PersonalAccount from './PersonalAccount';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
```

Comment:
we need to fetch for Donating Agency's primaryContact data for `AccountManager` section. Add `componentDidMount()`:
```
componentDidMount(): {
    let manager = this.props.account;
    if (manager.accountType === AccountType.DONATING_AGENCY_MEMBER && !manager.isAdmin) {
        // backend TODO: fetch DA primary contact data for AccountManager
        manager = {
            accountType: AccountType.DONATING_AGENCY_MEMBER,
            name: 'something',
            email: 'something',
            phone: 'something',
            position: 'something',
        };
    }
    this.setState({manager: manager});
}
```

Reply:
Added.